orchestrator-salt/src/main/resources/salt/salt/metadata/settings.sls

```diff
@@ -20,3 +22,5 @@
     'clusterName' : clusterName,
     'cluster_in_childenvironment' : cluster_in_childenvironment
 }) %}
+
+{% set hostattrs = salt['pillar.get']('hostattrs:'~grains['fqdn']) %}
```

Comment:
what is this `~` for?

Reply:
The ~ is apparently required when accessing variables inside of a pillar get. i.e. pillar.get(blah(grains.get))